modules/preview-react/InformationHighlight/lib/parts/Icon.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {SystemIcon, SystemIconProps, systemIconStencil} from '@workday/canvas-kit-react/icon';
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {cssVar} from '@workday/canvas-kit-styling';
+import {
+  infoIcon,
+  exclamationCircleIcon,
+  exclamationTriangleIcon,
+} from '@workday/canvas-system-icons-web';
+import {base} from '@workday/canvas-tokens-web';
+import {useInformationHighlightModel, Variant, Emphasis} from '../hooks/modelHook';
+
+export interface IconProps extends SystemIconProps {
+  variant?: Variant;
+  emphasis?: Emphasis;
+}
```

Comment:
This type extends `SystemIconProps` which always requires the `icon` prop. Since we're handling some defaults based on the `variant` we can clean up this type so that you don't have to set `icon={undefined}` when using the components. 

Also, `variant` and `emphasis` don't do anything because you're using the values from the `model` and not the props, so they should probably be removed.

I suggest doing something like this: 

```suggestion
export interface IconProps extends Omit<Partial<ExtractProps<typeof SystemIcon, never>>, 'icon'> {
  icon?: SystemIconProps['icon'];
}
```

- We omit `icon` from `SystemIcon`
- Make all props from `SystemIcon` optional
- Add back in `icon` as an optional prop types as the same value that SystemIcon expects.